src/dialogs/config-flow/step-flow-create-entry.ts

```diff
@@ -250,13 +266,15 @@ class StepFlowCreateEntry extends LitElement {
       await Promise.allSettled(entityUpdates);
     }
 
-    fireEvent(this, "flow-update", { step: undefined });
```

Comment:
This means we don't delete the previous flow now when we continue with the second flow, right?

Reply:
Moved all the code to the dialog for better cleanup but I doubt it improved anything. We don't need to delete the previous flow because it is finished if we get to a "create_entry" step.